plugins/node/opentelemetry-instrumentation-net/package.json

```diff
@@ -62,7 +62,7 @@
   },
   "dependencies": {
     "@opentelemetry/api": "^0.18.0",
-    "@opentelemetry/instrumentation": "^0.18.0",
-    "@opentelemetry/semantic-conventions": "^0.18.0"
+    "@opentelemetry/instrumentation": "^0.18.2",
```

Comment:
Please do not change the versions of the dependencies in individual packages. They are changed all at the same time

Reply:
Reverted